kbs/docker/Dockerfile

```diff
@@ -41,31 +41,25 @@ RUN cargo install --locked --path kbs/src/kbs --no-default-features --features c
 FROM ubuntu:22.04
 ARG ARCH=x86_64
 
+WORKDIR /tmp
+
 RUN apt-get update && \
     apt-get install -y \
-    clang \
     curl \
-    gnupg-agent
+    gnupg-agent && \
+    apt clean all && \
+    rm -rf /tmp/*
 
-# Install TDX Runtime Dependencies
+# Install Runtime Dependencies
 RUN if [ "${ARCH}" = "x86_64" ]; then curl -fsSL https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | \
-    gpg --dearmor --output /usr/share/keyrings/intel-sgx.gpg; fi
-RUN if [ "${ARCH}" = "x86_64" ]; then echo 'deb [arch=amd64 signed-by=/usr/share/keyrings/intel-sgx.gpg] https://download.01.org/intel-sgx/sgx_repo/ubuntu jammy main' | tee /etc/apt/sources.list.d/intel-sgx.list; fi
-RUN apt-get update
-RUN if [ "${ARCH}" = "x86_64" ]; then apt-get install -y --no-install-recommends \
+    gpg --dearmor --output /usr/share/keyrings/intel-sgx.gpg && \
+    echo 'deb [arch=amd64 signed-by=/usr/share/keyrings/intel-sgx.gpg] https://download.01.org/intel-sgx/sgx_repo/ubuntu jammy main' | tee /etc/apt/sources.list.d/intel-sgx.list && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
     libsgx-dcap-default-qpl \
     libsgx-dcap-quote-verify \
-    tpm2-tools; fi
-
-# Intel PCCS URL Configurations
```

Comment:
May I ask a rationale for removing this? I don't think the removal has nothing to do with reducing a image size (because it is just environment variable declaration and string replacement). Thanks!

Reply:
Yeah, it has nothing to do with the topic. Originally, I thought this code was actually useless. Most people who use KBS would mount this file instead of hardcoding it during the image building phase. I just deleted these lines by the way; Let me set this piece of content as a separate commit, and then collecte more opinions to see if we should keep it.